src/main/java/ro/secur/auth/service/UserService.java

```diff
@@ -88,8 +103,58 @@ public void changePassword(String username, ChangePasswordRequest request) {
         }
     }
 
+    public void forgotPassword(String email, HttpServletRequest request) {
+
+        String email1 = email.replace("{\"email\":{\"email\":\"", "");
+        String email2 = email1.replace("\"}}", "");
+        UserInfoEntity userInfoEntity = userInfoRepository.findByEmail(email2);
+        UserEntity userEntity = userRepository.findByUserInfoEntity(userInfoEntity);
+
+        if (userEntity != null) {
+            userEntity.setResetToken(UUID.randomUUID().toString());
+
+            save(userEntity);
```

Comment:
e posibil sa arunce o erroare saveu`? . Gen ca nu poate sa faca persist ?.  Eventual un handle de erroare daca e cazul.

Reply:
Poti adauga in ExceptionControllerAdvice in catch de errori generale de persist sau ce erroare ar aunca cand nu merge save`u